apps/bbsync/tests/test_integration.py

```diff
@@ -773,14 +775,16 @@ def test_affect_update(self, auth_client, test_api_uri):
             HTTP_BUGZILLA_API_KEY="SECRET",
             HTTP_JIRA_API_KEY="SECRET",
         )
+        print(response.content)
```

Comment:
There seems to be forgotten debugging `print`

Reply:
This was removed in a prior commit.